src/components/core/core.js

```diff
@@ -109,7 +106,10 @@ export default class Core extends UIObject {
    * @type {Object}
    */
   get activePlaybackEl() {
-    return this.activePlayback.$el.find('video')[0] || this.activePlayback.el
+    if (!this.activePlayback) return false
+    if (this.activePlayback.el)
+      return this.activePlayback.el
+    return this.activePlayback.$el.find('video')[0]
```

Comment:
To keep prioritizing the `find` on `$el` over the `activePlayback.el`

```suggestion
    if (!this.activePlayback) return null
    return this.activePlayback.$el.find('video')[0] || this.activePlayback.el
```

Reply:
What is the reason for prioritizing `find` over `$el.find` @jhonatangcavalcanti ?

I'm receiving `TypeError: Cannot read property 'find' of undefined` on broken tests this way.